.gitlab/fips_compliance/fips_compliance_e2e.yml

```diff
@@ -0,0 +1,31 @@
+---
+.fips-compliance-e2e:
+  stage: check_fips_compliance
```

Comment:
Could you please have the same name for the stage and the name of your folder, to keep it uniform with other stages?

Reply:
d00a364477a86a73e732e715379e4ce06c57fed1